shaders/src/light_indirect.fs

```diff
@@ -135,9 +135,13 @@ vec3 getSpecularDominantDirection(const vec3 n, const vec3 r, float roughness) {
 vec3 specularDFG(const PixelParams pixel) {
 #if defined(SHADING_MODEL_CLOTH)
     return pixel.f0 * pixel.dfg.z;
+#else
+#if defined(MATERIAL_HAS_SPECULAR_COLOR_FACTOR) || defined(MATERIAL_HAS_SPECULAR_FACTOR)
+    return mix(pixel.dfg.xxx, pixel.dfg.yyy, pixel.f0) * pixel.specular;
```

Comment:
You could always define pixel.specular to be the constant 1.0 when the extension is not in use and let the compiler no-op the multiplication (but this is fine as is).